logic.js

```diff
@@ -0,0 +1,47 @@
+// Modules
+
+const Tweeter = function () {
+    const _posts = []
+    let postIdCounter = _posts.length
+    let commentIdCounter = 0
+    const getPosts = () => _posts
+    const addPost = (pText) => {
+        _posts.push({ text: pText, id: "p" + (postIdCounter + 1), comments: [] })
+        postIdCounter++
+    }
+    const removePost = (postID) => {
+        for (p of _posts) {
+            if (p.id == postID) {
+                _posts.splice(_posts.indexOf(p), 1)
+            }
+        }
+    }
+    const addComment = (cText, postID) => {
+        for (p of _posts) {
+            if (p.id == postID) {
```

Comment:
Use `===` as your default comparison. We like to be as strict as possible and only use more lenient things like `==` when we really need to.